kafka-plugins-0.10/src/main/java/io/cdap/plugin/alertpublisher/KafkaAlertPublisher.java

```diff
@@ -167,11 +168,11 @@ private void validate() {
       try {
         Topic.validate(topic);
       } catch (InvalidTopicException e) {
-        throw new IllegalArgumentException(String.format("Topic name %s is not a valid kafka topic. Please provide " +
-                                                           "valid kafka topic name. %s", topic, e.getMessage()));
+        collector.addFailure(String.format("Topic name %s is not a valid kafka topic. Please provide " +
+                                                           "valid kafka topic name. %s", topic, e.getMessage()), null);
```

Comment:
Should be including the name of the property as a cause for the failure.
```
.withConfigProperty("topic")
```

This is what allows the UI to highlight the topic and display the error next to the topic box.

See https://github.com/data-integrations/google-cloud/blob/release/0.13/src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySourceConfig.java#L142 for an example.